ateam_ai/src/behavior/behavior_executor.cpp

```diff
@@ -22,8 +22,10 @@
 
 #include "behavior/behavior_feedback.hpp"
 
-BehaviorExecutor::BehaviorExecutor(BehaviorRealization & behavior_realization)
-: behavior_realization(behavior_realization) {}
+BehaviorExecutor::BehaviorExecutor(
+  BehaviorRealization & behavior_realization,
+  rclcpp::Publisher<ateam_msgs::msg::RobotMotionCommand>::SharedPtr robot_commands)
+: behavior_realization(behavior_realization), robot_commands(robot_commands) {}
```

Comment:
I'm don't super liking trying to pass down the motion command pubs, I think I might change this to return an array of motion commands (or even just return a list of ros msgs), and have a separate module that converts motion commands to the ros msg. I'll do that in another PR to clean this up though